Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rail
 gem 'jsonapi-serializer', '~> 2.2' # https://rubygems.org/gems/jsonapi-serializer
 gem 'pg', '~> 1.1' # https://rubygems.org/gems/pg
 gem 'puma', '~> 5.0' # https://rubygems.org/gems/puma
+gem 'discard', '~> 1.2' #https://github.com/jhawthorn/discard
```

Comment:
you need to place the gems in alphabetical order

Reply:
done